bp_common/test/cfg/sample.cfg

```diff
@@ -4,7 +4,7 @@
   "bios": "prog.mainram",
   "load": "prog",
   "memory_base_addr": 0x80000000,
-  "memory_size": $SAMPLE_MEMSIZE,
```

Comment:
Can we keep this parameter? 256 is a large startup overhead for small samples

Reply:
I'll add the SAMPLE_MEMSIZE flag back in the Makefiles(for smaller checkpoint files). But just to clarify, the setting here is for initializing the Dromajo instance in cosimulation. Which has no overhead and there is no point in it being smaller than the memory size.

Reply:
The dromajo checkpoint memory size == the mem checkpoint size, isn’t it? So the nbf loader takes longer?

Reply:
Yes, if we're loading in the memory with nbf loader.